fluster/main.py

```diff
@@ -81,8 +97,9 @@ def run(self) -> None:
                 test_suites_dir=args.test_suites_dir,
                 decoders_dir=self.decoders_dir,
                 resources_dir=args.resources,
-                results_dir=args.output,
+                output_dir=args.output,
                 use_emoji=not args.no_emoji,
+                verbose=args.verbose if "verbose" in args else False,
```

Comment:
@ylatuya @mdimopoulos 

Verbose should be a global argument. But this change breaks the BS.